Application.Tests/Mappings/SchoolDatumTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Application.Mapping;
+using Domain.Constants;
+using FluentAssertions;
+using Xunit;
+
+namespace Application.Tests.Mappings;
+public class SchoolDatumTests
+{
+    [Theory]
+    [InlineData(26)]
```

Comment:
I think it's worth having these as constants as in making a start on building up the EstablishmentType list so that it's more clear what the test is asserting. This should be used in the `IsValidForService()` method which would allow the comment there to be removed.

Reply:
Changed as requested